### ISSUE TITLE
Add `jax.numpy` namespace functions to Jax frontend

### ISSUE THREAD

**Issue by hirwa-nshuti** [unifyai/ivy#4612]
Add [`jax.numpy` namespace functions](https://jax.readthedocs.io/en/latest/jax.numpy.html) to Jax frontend

- [ ] ndarray.at
- [x] #4884
- [x] #4915
- [x] #4916
- [x] #4917
- [x] #4897
- [x] #6902
- [ ] #7046
- [x] #7047
- [ ] angle
- [x] #6651
- [ ] append
- [ ] apply_along_axis
- [ ] apply_over_axes
- [x] #7322
- [x] #5248
- [x] #5249
- [x] #5250
- [x] #5251
- [x] #5369
- [x] #5371
- [x] #6895
- [x] #5214
- [x] #6091
- [x] #7423
- [x] #7425
- [ ] around
- [x] #7210
- [x] #5700
- [x] #5867
- [ ] array_repr
- [ ] array_split
- [ ] array_str
- [x] #7712
- [ ] atleast_1d
- [ ] atleast_2d
- [ ] atleast_3d
- [ ] average
- [ ] bartlett
- [x] #6163
- [x] #5857
- [x] #6211
- [x] #5925
- [x] #6235
- [ ] blackman
- [ ] block
- [ ] bool_
- [ ] broadcast_arrays
- [ ] broadcast_shapes
- [x] #4949
- [ ] c_
- [x] #7770
- [ ] cbrt
- [ ] cdouble
- [x] #6326
- [ ] character
- [ ] choose
- [x] #4957
- [ ] column_stack
- [ ] complex_
- [ ] complex128
- [ ] complex64
- [ ] complexfloating
- [ ] complexWarning
- [ ] compress
- [x] #4948
- [ ] conj
- [ ] conjugate
- [ ] convolve
- [ ] copy
- [ ] copysign
- [ ] corrcoef
- [ ] correlate
- [x] cos
- [ ] cosh
- [ ] count_nonzero
- [ ] cov
- [ ] cross
- [ ] csingle
- [x] #6258
- [x] #6465
- [x] #6466
- [x] #6467
- [ ] #7477
- [ ] delete
- [x] #6655
- [ ] diag_indices
- [ ] diag_indices_from
- [ ] #6893
- [x] #7429
- [ ] diff
- [ ] digitize
- [x] #7771
- [ ] divmod
- [x] #4946
- [ ] double
- [ ] dsplit
- [ ] dstack
- [ ] dtype
- [ ] ediff1d
- [ ] #4950
- [ ] einsum_path
- [ ] empty
- [ ] empty_like
- [ ] #7050
- [ ] exp
- [x] #6471
- [ ] expand_dims
- [ ] expm1
- [ ] extract
- [ ] #7354
- [ ] fabs
- [ ] finfo
- [ ] fix
- [ ] flatnonzero
- [ ] flexible
- [x] #6658
- [x] #6660
- [x] #6028
- [ ] float_
- [x] #6321
- [ ] float16
- [ ] float32
- [ ] float64
- [ ] floating
- [x] #5691
- [ ] floor_divide
- [x] #5705
- [ ] fmin
- [ ] fmod
- [ ] frexp
- [ ] frombuffer
- [ ] fromfile
- [ ] fromfunction
- [ ] fromiter
- [ ] fromstring
- [ ] from_dlpack
- [ ] full
- [ ] full_like
- [x] #6532
- [ ] generic
- [ ] geomspace
- [ ] get_printoptions
- [ ] gradient
- [ ] #7051
- [x] #7052
- [ ] hamming
- [ ] hanning
- [x] #6458
- [ ] histogram
- [ ] histogram_bin_edges
- [ ] histogram2d
- [ ] histogramdd
- [ ] hsplit
- [x] #6751
- [ ] hypot
- [x] #6561
- [ ] identity
- [ ] iinfo
- [ ] imag
- [ ] in1d
- [ ] index_exp
- [ ] indices
- [ ] inexact
- [ ] inner
- [ ] insert
- [ ] int_
- [ ] #7677
- [ ] int32
- [ ] int64
- [ ] int8
- [ ] integer
- [ ] interp
- [ ] intersect1d
- [ ] invert
- [ ] isclose
- [ ] iscomplex
- [ ] iscomplexobj
- [ ] isfinite
- [ ] isin
- [ ] isinf
- [ ] isnan
- [x] #6572
- [x] #6577
- [ ] isreal
- [ ] isrealobj
- [ ] isscalar
- [ ] issubdtype
- [ ] issubsctype
- [ ] iterable
- [ ] ix_
- [ ] kaiser
- [x] #6584
- [x] #6599
- [ ] ldexp
- [ ] left_shift
- [ ] #7053
- [ ] #7054
- [ ] lexsort
- [ ] linspace
- [ ] load
- [ ] log
- [x] #7277
- [x] #5895
- [ ] #5896
- [x] #7286
- [x] #6602
- [ ] logical_and
- [ ] logical_not
- [ ] logical_or
- [ ] logical_xor
- [ ] logspace
- [ ] mask_indices
- [ ] matmul
- [x] #7049
- [x] #6901 
- [x] #4953
- [ ] median
- [ ] meshgrid
- [ ] mgrid
- [ ] #7048
- [ ] #6900
- [ ] mod
- [ ] modf
- [x] #5989
- [x] #6898
- [x] #6899
- [ ] nan_to_num
- [ ] nanargmax
- [ ] nanargmin
- [ ] nancumprod
- [ ] nancumsum
- [ ] nanmax
- [ ] nanmean
- [ ] nanmedian
- [ ] nanmin
- [ ] nanpercentile
- [ ] nanprod
- [ ] nanquantile
- [ ] nanstd
- [ ] nansum
- [ ] nanvar
- [ ] ndarray
- [ ] ndim
- [x] #7711
- [ ] nextafter
- [ ] nonzero
- [x] #7055
- [ ] number
- [ ] object_
- [ ] ogrid
- [x] #7857
- [x] #7856
- [ ] outer
- [ ] packbits
- [ ] pad
- [ ] percentile
- [ ] piecewise
- [ ] poly
- [ ] polyadd
- [ ] polyder
- [ ] polydiv
- [ ] polyfit
- [ ] polyint
- [ ] polymul
- [ ] polysub
- [ ] polyval
- [ ] positive
- [x] #6082
- [ ] printoptions
- [ ] prod
- [ ] product
- [ ] promote_types
- [ ] ptp
- [ ] quantile
- [ ] r_
- [x] #7749
- [ ] radians
- [ ] ravel
- [ ] ravel_multi_index
- [ ] real
- [ ] reciprocal
- [ ] remainder
- [ ] repeat
- [x] #4952
- [ ] resize
- [ ] result_type
- [ ] right_shift
- [ ] rint
- [ ] roll
- [ ] rollaxis
- [ ] roots
- [ ] #6795
- [ ] round
- [ ] round_
- [ ] row_stack
- [ ] s_
- [ ] save
- [ ] savez
- [ ] searchsorted
- [ ] select
- [ ] set_printoptions
- [ ] setdiff1d
- [ ] setxor1d
- [ ] shape
- [ ] sign
- [ ] signbit
- [ ] signedinteger
- [x] #6896
- [ ] sinc
- [ ] single
- [x] #6897
- [ ] size
- [x] #7023
- [ ] sort
- [ ] sort_complex
- [ ] split
- [x] #7854
- [ ] square
- [ ] #7427
- [x] #7710
- [ ] std
- [ ] #5300
- [x] #6317
- [ ] swapaxes
- [x] #7709
- [ ] take_along_axis
- [x] #7165
- [x] #7166
- [x] #7755
- [ ] tile
- [ ] trace
- [x] #7855
- [x] #6618
- [ ] tri
- [ ] tril
- [ ] tril_indices
- [ ] tril_indces_from
- [ ] trim_zeros
- [ ] triu
- [ ] triu_indices
- [ ] triu_indices_from
- [x] #7776
- [x] #6330
- [ ] uint
- [x] #5130
- [ ] unint32
- [ ] uint64
- [ ] uint8
- [ ] union1d
- [ ] unique
- [ ] unpackbits
- [ ] unravel_index
- [ ] unsignedinteger
- [ ] unwrap
- [ ] vander
- [x] #4951
- [ ] vdot
- [ ] vectorize
- [ ] vsplit
- [x] #6732
- [ ] where
- [x] #5630
- [x] #7449

**Comment by mostfa29** [comment]
#5000

**Comment by mostfa29** [comment]
#5000

**Comment by trihernandez** [comment]
- [ ] #5130

**Comment by kojomensahonums** [comment]
#5403

**Comment by AlphaPro06** [comment]
Adding these jax.numpy namespace functions to Jax frontend:

- [ ] bitwise_and #4612
- [ ] bitwise_not #4612
- [ ] bitwise_or #4612
- [ ] bitwise_xor #4612

**Comment by kojomensahonums** [comment]
Hi,
Can I create multiple issues at once, like I reserve all these tasks for
myself at once?

Thanks,
Andrew

**Comment by Change-m** [comment]
deg2rad #5560

**Comment by Change-m** [comment]
rad2deg #5561

**Comment by samuelbido** [comment]
#5630

**Comment by Torus403** [comment]
Arccos: https://github.com/unifyai/ivy/issues/5248 
Arccosh: https://github.com/unifyai/ivy/issues/5249
Arcsin: https://github.com/unifyai/ivy/issues/5250
Arcsinh: https://github.com/unifyai/ivy/issues/5251

**Comment by ema-alb** [comment]
- [ ] #5691

**Comment by JaithraVarma** [comment]
#5700

**Comment by AlphaPro06** [comment]
cumsum #5682

**Comment by AlphaPro06** [comment]
cumsum #5681

**Comment by AlphaPro06** [comment]
asarray #5680

**Comment by ukj02** [comment]
#5832

**Comment by AlphaPro06** [comment]
#5857

**Comment by JaithraVarma** [comment]
#5867

**Comment by AlphaPro06** [comment]
- [ ] #5925

**Comment by nassimberrada** [comment]
moveaxis #5989

**Comment by nassimberrada** [comment]
moveaxis #5989

**Comment by nassimberrada** [comment]
flipud #6028

**Comment by waseef3277** [comment]
argwhere #6049

**Comment by Pranjal-bisht** [comment]
- [] #6211

**Comment by Pranjal-bisht** [comment]
- [] #6235

**Comment by Nunyapa** [comment]
#6317

**Comment by nassimberrada** [comment]
float_power #6321

**Comment by Oluwadurotimi10** [comment]
- [ ] #6326

**Comment by Oluwadurotimi10** [comment]
- [ ] #6330

**Comment by nassimberrada** [comment]
heaviside #6458

**Comment by hirwa-nshuti** [comment]
- [ ]  #6465

**Comment by hirwa-nshuti** [comment]
- [ ] #6466

**Comment by nassimberrada** [comment]
exp2 #6471

**Comment by Change-m** [comment]
vstack #6732

**Comment by Change-m** [comment]
rot90 #6795

**Comment by Change-m** [comment]
diagflat #6893

**Comment by hirwa-nshuti** [comment]
- [ ] #6895

**Comment by hirwa-nshuti** [comment]
- [ ] #6896

**Comment by hirwa-nshuti** [comment]
- [ ] #6897

**Comment by hirwa-nshuti** [comment]
- [ ] #6898

**Comment by hirwa-nshuti** [comment]
- [ ] #6899

**Comment by hirwa-nshuti** [comment]
- [ ] #6900

**Comment by hirwa-nshuti** [comment]
- [ ] #6901

**Comment by hirwa-nshuti** [comment]
- [ ] #6902

**Comment by hirwa-nshuti** [comment]
- [ ] #7023

**Comment by hirwa-nshuti** [comment]
- [ ] #7046

**Comment by hirwa-nshuti** [comment]
- [ ] #7047

**Comment by hirwa-nshuti** [comment]
- [ ] #7048

**Comment by hirwa-nshuti** [comment]
- [ ] #7049

**Comment by hirwa-nshuti** [comment]
- [ ] #7050

**Comment by hirwa-nshuti** [comment]
- [ ] #7051

**Comment by hirwa-nshuti** [comment]
- [ ] #7052

**Comment by hirwa-nshuti** [comment]
- [ ] #7053

**Comment by hirwa-nshuti** [comment]
- [ ] #7054

**Comment by hirwa-nshuti** [comment]
- [ ] #7055

**Comment by Taciturny** [comment]
- [ ] amax

**Comment by Taciturny** [comment]
- [ ] #7086

**Comment by maciejbalawejder** [comment]
#7105

**Comment by ndurumo254** [comment]
#7147

**Comment by TAUIL-Abd-Elilah** [comment]
#7164

**Comment by TAUIL-Abd-Elilah** [comment]
#7165

**Comment by TAUIL-Abd-Elilah** [comment]
#7166

**Comment by danielmunioz** [comment]
#7264 7264

**Comment by danielmunioz** [comment]
#7264

**Comment by danmerus** [comment]
[ ] #7322

**Comment by TalhaArshad3** [comment]
- [ ] #7354

**Comment by TalhaArshad3** [comment]
#7354

**Comment by karalleyna** [comment]
- [ ] #7423

**Comment by karalleyna** [comment]
- [ ] #7425

**Comment by karalleyna** [comment]
- [ ] #7427

**Comment by karalleyna** [comment]
- [ ] #7429

**Comment by karalleyna** [comment]
- [ ] #7444

**Comment by karalleyna** [comment]
- [ ] #7445

**Comment by karalleyna** [comment]
- [ ] #7446

**Comment by karalleyna** [comment]
- [ ] #7447

**Comment by karalleyna** [comment]
- [ ] #7448

**Comment by karalleyna** [comment]
- [ ] #7449

**Comment by ndurumo254** [comment]
#7477

**Comment by D0m-inic** [comment]
int16 #7677

**Comment by ndurumo254** [comment]
- [ ]#7477

**Comment by ndurumo254** [comment]
- [ ] #7147

**Comment by hirwa-nshuti** [comment]
- [ ] #7771

**Comment by hirwa-nshuti** [comment]
- [ ] #7772

**Comment by hirwa-nshuti** [comment]
- [ ] #7776

**Comment by swaingotnochill** [comment]
- [ ] #9350

**Comment by thegeek13242** [comment]
int32 #10734

**Comment by H4ckOm** [comment]
- [ ] #13121

**Comment by ShreyanshBardia** [comment]
#22348
